auth/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+import com.android.build.gradle.internal.dsl.TestOptions
+
+android {
+    buildTypes {
+        getByName("release") {
+            postprocessing {
+                consumerProguardFiles("auth-proguard.pro")
+            }
+        }
+    }
+
+    lintOptions {
+        disable("UnusedQuantity")
+        disable("UnknownNullness") // Too much to deal with right now
```

Comment:
Fine with the disabling but can you explain what's going on?

Reply:
Yeah, I realize the nullability changes account for a lot more thrashing than originally anticipated. I've rebased those out and will make another PR with a better explanation. (As a short explanation, Tor is continuing his eternal quest to keep us on our toes with new lint checks. 😆)